components/sensor/tof10120.rst

```diff
@@ -0,0 +1,52 @@
+TOF10120 Time Of Flight Distance Sensor
+=======================================
+
+.. seo::
+    :description: Instructions for setting up TOF10120 distance sensors in ESPHome.
+    :image: vl53l0x.jpg
+    :keywords: VL53L0X
```

Comment:
this SEO will show the wrong sensor image

Reply:
🤦

Reply:
fixed